src/server/config/swagger.json

```diff
@@ -9,6 +9,98 @@
   "host": "",
   "basePath": "/api",
   "paths": {
+    "/batch-default-values/{batchId}": {
```

Comment:
Did we add this route? or is it legacy code? I think we can remove it.

Reply:
This is another endpoint :)

Reply:
Ah. I see it [here](https://github.com/HackYourFuture-CPH/FP-class11/blob/348853861cc9192bdacb39d89ae536be820f7f50/src/server/api/routes/api-router.js#L35). It was just added to swagger in this PR. My bad 😓